### PR TITLE
eth/*: golint updates for this or self warning

### PR DESCRIPTION
**Pull request by kielbarry**
In the eth directory, I have updated the variable names for receivers generating a golint warning regarding use of generic names like this or self.

### CODE REVIEW

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 6505abf73dfe51673a0e564cc5d6f2b6c04d692f
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

File: eth/handler.go [hunk 1]
```diff
@@ -760,13 +760,13 @@ type NodeInfo struct {
...
-		Genesis:    self.blockchain.Genesis().Hash(),
-		Config:     self.blockchain.Config(),
+		Network:    pm.networkId,
+		Difficulty: pm.blockchain.GetTd(currentBlock.Hash(), currentBlock.NumberU64()),
+		Genesis:    pm.blockchain.Genesis().Hash(),
+		Config:     pm.blockchain.Config(),
 		Head:       currentBlock.Hash(),
 	}
-}
+}
```
Comment (karalabe) [inline eth/handler.go]: Please add back the newline.

File: eth/backend.go [hunk 2]
```diff
@@ -325,13 +325,13 @@ func (s *Ethereum) Etherbase() (eb common.Address, err error) {
 	return common.Address{}, fmt.Errorf("etherbase must be explicitly specified")
 }
 
-// set in js console via admin interface or wrapper from cli flags
-func (self *Ethereum) SetEtherbase(etherbase common.Address) {
-	self.lock.Lock()
-	self.etherbase = etherbase
-	self.lock.Unlock()
+// SetEtherbase in js console via admin interface or wrapper from cli flags
```
Comment (karalabe) [inline eth/backend.go]: > SetEtherbase sets the mining reward address.

**Comment by GitCop** [comment]
Thank you for your contribution! Your commits seem to not adhere to the repository coding standards

 * Commit: 6505abf73dfe51673a0e564cc5d6f2b6c04d692f
  * Commits must be prefixed with the package(s) they modify

Please check the [contribution guidelines](https://github.com/ethereum/go-ethereum#contribution) for more details.

---
This message was auto-generated by https://gitcop.com

**Pull request merged**